test/evp_extra_test.c

```diff
@@ -2571,66 +2571,1014 @@ static int test_EVP_SM2(void)
 
 #endif
 
+typedef unsigned char byte;
```

Comment:
uint8 might be more typical

Reply:
The original code used ordinary (signed) chars, I just changed to unsigned chars, but did not want to introduce new use of `uint8_t`, and some hypothetical compiler could get picky casting literal string to `uint8_t`.

Reply:
IMO we should either use `unsigned char` or `uint8_t` for bytes. Historically we always used `unsigned char`, but in more recent times we seem to be often using `uint8_t`. I don't think we should introduce a new `byte` type as yet another alternative to represent bytes. I see no reason why a compiler would complain about this.

Reply:
Changed to `uint8_t` as requested.